CHANGELOG.md

```diff
@@ -7,13 +7,34 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [5.3.0] - 2019-03-07
+
+### Added
+- Additional additional check config and entity information to event details page.
+- fixed all known TLS vulnerabilities affecting the backend server:
```

Comment:
This is gonna sound so annoying but could we make the capitalization consistent? 😅